summarizer_server/server.py

```diff
@@ -151,6 +160,30 @@ def configure_logger(debug):
     app.logger.propagate = True
 
 
+def send_metric_summarize(email):
+    dd_api_key = os.environ.get("DD_API_KEY", "")
+    host = os.environ.get("HOST", "")
+    userhash = "unknown"
+    if email:
+        userhash = hashlib.sha256(bytes(email, "UTF")).hexdigest()
```

Comment:
I think instead of hashing the email here, we should hash it on the client-side and verify a valid hash on the server side. That way we would have absolutely no idea what the original email was.

Reply:
Even if we hash it on the frontend you can find the original by doing brute force. The only way to prevent that is to hash the email + salt, but we don't have anywhere to keep the salt since then we'd have to store things in our own db. I guess if we hash it on the frontend we are able to tell users we don't send the plain text email on summarize requests.

Reply:
I see, if we are hashing i think we can store the salt locally within the Google Chrome storage? But after discussion, I think a good approach would be just not keeping the email at all and just have a tally of the API call counts.

Reply:
ok done, now it's only call counts